### DISCUSSION TITLE
Why is v4 the default docs if it's still in beta?

### DISCUSSION THREAD

**Discussion by lindeberg** [nextauthjs/next-auth#3035] [Help] [answered]
Why is v4 the default docs if it's still in beta?

**Comment by balazsorban44** [comment, marked as answer]
because we would like people to try it out and give feedback, also we moved the docs to its own repository, and set up a new deployment, it felt appropriate to do it at the same time.

Also all effort on v3 has stopped, so it is in practice "complete" now.


https://github.com/nextauthjs/next-auth/tree/main/www -> https://github.com/nextauthjs/docs
